### ISSUE TITLE
Detail MVP & Post MVP in README

### ISSUE THREAD

**Issue by dominik-ilja** [Dominik-Ilja/solar-conveyance#15]
Project Requirements :

- MVP: Your goals for MVP, including the minimum, need-to-have features of your app.
- Post-MVP: Write out what your goals are for post-MVP, including nice-to-have features that you would like to implement once your MVP is complete. (If you plan to use a component library, please link that here as well.)
